Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -18,19 +18,52 @@
 #import "ABContactInternal.h"
 #import "UWP/WindowsApplicationModelContacts.h"
 
+@interface __ABContactOperation : NSObject
+
+@property BOOL shouldDelete;
+@property WACContact* contact;
+
+- (id)initWithContact:(WACContact*)contact shouldDelete:(BOOL)shouldDelete;
+
+@end
+
+@implementation __ABContactOperation
+
+- (id)initWithContact:(WACContact*)contact shouldDelete:(BOOL)shouldDelete {
+    if (self = [super init]) {
+        self.shouldDelete = shouldDelete;
+        self.contact = contact;
+    }
+
+    return self;
+}
+
+@end
+
 @interface _ABAddressBookManager ()
 
+- (void)_setError:(CFErrorRef*)error message:(NSString*)message;
```

Comment:
>_setError [](start = 8, length = 9)

Just curious why this method is declared here instead of in the header file's interface declaration.